### DISCUSSION TITLE
Preparing for 2.3.120

### DISCUSSION THREAD

**Discussion by Sahale** [Security-Onion-Solutions/securityonion#7606] [Unsupported Versions] [answered]
Hi.
After problems in upgrading 2.3.100 to 2.3.110 I decided not to upgrade prod environment and wait for 2.3.120. According to #7545 I need to modify elasticsearch templates to new format. I modified them in test environment and it works good.
But I want to understand how to prepare for .120 in prod environment to minimise downtime. Is this plan correct:

1. While on .100 delete templates from `/opt/so/saltstack/local/pillar/global.sls`;
2. Create new index templates, remove  `/opt/so/saltstack/local/pillar/elasticsearch/manager.sls`/`search.sls` config files;
3. Start soup;
4. Delete old templates.

Or I missed something?
Thanks!

**Comment by weslambert** [comment, marked as answer]
You shouldn't need to delete anything once you update to .120. You'll just need to specify your custom templates in the appropriate place (will be documented in the .120 release notes), and the reference to their curator settings or any custom index settings in the pillar.
